tsconfig.json

```diff
@@ -6,7 +6,8 @@
       "es2015.core",
       "es2015.collection",
       "es2015.iterable",
-      "es2016.array.include" // Supported by Node 6+
+      "es2016.array.include", // Supported by Node 6+
```

Comment:
Doesn't this conflict with the target of "es2015"? As far as I can tell, we have no Babel in this project.

Reply:
I've removed the *dom* lib from tsconfig, this was a mistake. It was required for prismjs, but not necessary for shiki.